CFP reminder post (#2068)
writethedocs/www: docs/conf/portland/2024/news/cfp-reminder.rst

:template: {{year}}/generic.html

> .. post:: Jan 03, 2024

Author: ```suggestion
.. post:: Jan 04, 2024
```

>    :tags: {{shortcode}}-{{year}}, website, cfp, tickets

Reviewer: ```suggestion
   :tags: {{shortcode}}-{{year}}, cfp, tickets, grants
```

Call for Proposals and Grant Application Reminder
===================================================

Happy New Year, fellow documentarians!

This is your scheduled reminder to kick off 2024 by `submitting a talk <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`__ to Write the Docs Portland! You can do so until **{{ cfp.ends }}**.

> In other conference news, `tickets are on sale <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_, and you can still `apply for an opportunity grant <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_, read on for more information on any of that good stuff.

Reviewer: ```suggestion
In other conference news, `tickets are on sale <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_, and you can still `apply for an opportunity grant <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_, read on for more information.
```

Author: :-D ok ok

you can always ask for talk feedback on the `Write the Docs Slack <https://www.writethedocs.org/slack/>`_.
The community is there to help!

The most important dates to remember:

* Talk proposals are due by {{ cfp.ends }}.
* Everyone who submits will get a response by {{ cfp.notification }}.

.. image:: /_static/img/speakers.jpeg
   :width: 400
   :alt: Ashleigh Rentz speaking at WTD in 2023

.. raw:: html

    <br /><p />

Get your ticket for the conference
----------------------------------

**The Portland conference will be held on {{ date.short }}**, in Revolution Hall.

The `full event website <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/>`_ is live and already includes many details. We'll be updating it with more information as we get closer to the event.

If you're excited to join us again in person,
go ahead and `register for your ticket now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_!
We are still planning to limit capacity of the event to ensure that we have a safe and community-focused experience for everyone,
so we expect tickets to sell out a few weeks before the conference.

> Similar to our past conferences, we're going to keep as much of the spirit of the event as possible:

Reviewer: ```suggestion
Similar to our past conferences, we're going to have a wide range of ways to engage:
```

- Speaker Presentations
- Writing Day
- Unconference Sessions
- Sponsor Expo + Job Fair
- Welcome Reception
- Conference Party

We're excited to see you all in Portland again soon.

Opportunity Grants
------------------

> Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until    **{{ grants.ends }}**.

Reviewer: ```suggestion
Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until **{{ grants.ends }}**.
```

> View our `Opportunity Grants page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ for information and the grant application, .

Reviewer 2: ```suggestion
View our `Opportunity Grants page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ for information and the grant application.
```

Support our community by sponsoring
-----------------------------------

Every year we are fortunate to have the support of like-minded organizations and communities,

> who believe in the collaborative and welcoming community spirit and help us make our events great.

Reviewer: ```suggestion
who believe in the collaborative and welcoming community spirit and help us make our events great.

Thanks so much to the following companies for supporting the conference this year:

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-config.yaml
   :template: {{year}}/sponsors-simplelist.rst
```

We should be listing sponsors in each email... probably makes sense to add this to the template or something, but baby steps :)
